test/test.js

```diff
@@ -223,10 +223,10 @@ test('uses the log level passed in as an option, where the level is a custom one
   const logger = pinoHttp(
     {
       customLevels: {
-        custom: 25
+        infoCustom: 25
       },
-      useLevel: 'custom',
-      level: 'custom'
+      useLevel: 'infoCustom',
+      level: 'infoCustom'
```

Comment:
If needed, I could duplicate the test for the lowercase scenario too. I'm assuming here that both cases are covered by this test now.